odoo/addons/base/models/ir_cron.py

```diff
@@ -393,36 +393,46 @@ def toggle(self, model, domain):
         return self.try_write({'active': active})
 
     @api.model
-    def _trigger(self, delay=None, at=None):
+    def _trigger(self, at=None):
```

Comment:
I find the new `_trigger_multi` a bit "clunky".
We could accept  either a datetime or a list for `at`, as it is very simple and intuitive to use:
```python
cron._trigger(record.schedule_date)
cron._trigger(records.mapped('schedule_date'))
```
The argument against it is that it makes overriding tricky.
But this is a low-level mechanism, why would you override `_trigger`?

And the current design is not bug-free either: if you override `_trigger`, you are in deep :hankey: because you should have overridden `_trigger_multi`!

Reply:
I have `write` override ptsd

Reply:
Frankly your call on this one, tde wanted `_trigger` to support both a unique datetime and a list of datetimes too. You better know how to write APIs then I do, my POV is that if one wants to override `_trigger` he will read the method and figure out he must override `_trigger_multi` instead.

Reply:
Also, reading the mapped I should replace False by now() in the list

Reply:
This is not correct: a `False` coming from `records.mapped('schedule_date`) means to schedule nothing for a given record.  In this case, the caller must ensure that the list only contains datetime objects:
```python
cron._trigger(filter(bool, records.mapped('schedule_date')))
```